aws/amazon-q-eclipse
# Adds display static mock for test

*Issue #, if available:*
When the IDE first starts up, the plugin will try to attach a document listener to the active document, which enables auto triggers. 
However, we do not have a similar logic to attach a listener to the part reference which contains the editors. As a result, nothing is attaching document listeners to new documents because the listener that is responsible for doing so is itself not attached. 
This PR addresses this problem by introducing similar logic that was used to attach the document listener. 

*Description of changes:*
- Adds timer exec on display to look for active `IPartService` to attach part listener. 

*Test:*
- Open IDE
- Switch to a file that was not previously opened / focused on
- Type and observe auto trigger working

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
